README.md

```diff
@@ -0,0 +1,33 @@
+# CWL Novice tutorial template
+
+CWL Template for the [CWL Novice tutorial](https://carpentries-incubator.github.io/cwl-novice-tutorial/index.html).
+
+This template contains some of the scripts used in the tutorial. 
+It also contains instructions for installing the bio-cwl-tools library.
+
+Before following the tutorial you will need to install [VSCode](https://code.visualstudio.com/), [docker](https://www.docker.com/), and [cwltool](https://github.com/common-workflow-language/cwltool). You will also need a working python installation, and either `pip` or `conda`. See the course [Setup page](https://carpentries-incubator.github.io/cwl-novice-tutorial/setup.html) for more details.
+
+ 
+#### Setup Instructions
+
+This template should be used to create a new git repository. The steps to do this are:
```

Comment:
Fair Q.

I think it would be useful for learners to have a repository space that they can record their work in, for later reference. But perhaps we make this optional - and say that they can either create their own repository, or directly pull from this repository.

We do definitely need them to use git for the checkout though - as the library submodule won't be setup if they simply download a zip of the directory.

Reply:
I've added alternate setup instructions, for direct cloning of the repository.